src/PeclEvLoop.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+namespace React\EventLoop;
+
+use Ev;
+use EvLoop;
+use React\EventLoop\Tick\FutureTickQueue;
+use React\EventLoop\Timer\Timer;
+use React\EventLoop\Timer\TimerInterface;
+use SplObjectStorage;
+
+/**
+ * @see https://bitbucket.org/osmanov/pecl-ev/overview
+ */
+class PeclEvLoop implements LoopInterface
+{
+    private $loop;
+    private $futureTickQueue;
+    private $timers;
+    private $readStreams = [];
+    private $writeStreams = [];
+    private $running;
+
+    public function __construct()
+    {
+        $this->loop             = new EvLoop();
+        $this->futureTickQueue  = new FutureTickQueue($this);
+        $this->timers           = new SplObjectStorage();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addReadStream($stream, callable $listener)
```

Comment:
@jsor, ok, thank you for the explanation 😃 , I agree with your arguments (but I just looked through `LibEvLoop`, `StreamSelectLoop` and other loops sources and there are a lot of "only `@inheritdoc`" PHPDocs).

I removed unnecessary `@inheritdoc` blocks from `PeclEvLoop` in 7128aaf.

Btw, I think that it'll be better to cleanup the history of this PR before merging it into master  (cause of a lot of little fix commits). I will do it tomorrow morning (UTC).